templates/opensshd.conf.j2

```diff
@@ -97,6 +97,11 @@ UsePAM {{ 'yes' if (ssh_use_pam|bool) else 'no' }}
 AuthenticationMethods publickey,keyboard-interactive
 {% endif %}
 
+# Force public key auth then ask for pam device input
+{% if ssh_pam_device %}
+AuthenticationMethods publickey,keyboard-interactive:pam
```

Comment:
We have almost the same configuration on lines 96-98, except for the `:pam`-part. What's the difference between these two? Are both needed?

Reply:
Thanks for your explanation!